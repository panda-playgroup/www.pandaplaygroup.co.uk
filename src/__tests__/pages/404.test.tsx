import { render } from "@testing-library/react";
import NotFoundPage, { Head } from "../../pages/404";

describe("not found", () => {
    it("renders", () => {
        const { asFragment } = render(<NotFoundPage />);

        expect(asFragment()).toMatchSnapshot();
    });
});

describe("head", () => {
    it("renders", () => {
        const { asFragment } = render(<Head />);

        expect(asFragment()).toMatchSnapshot();
    });
});
