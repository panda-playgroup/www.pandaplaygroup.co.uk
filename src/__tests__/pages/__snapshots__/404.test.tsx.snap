// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`head renders 1`] = `
<DocumentFragment>
  <title>
    Not found
  </title>
</DocumentFragment>
`;

exports[`not found renders 1`] = `
<DocumentFragment>
  <main
    style="color: rgb(35, 33, 41); padding: 96px; font-family: -apple-system, Roboto, sans-serif, serif;"
  >
    <h1
      style="margin-top: 0px; margin-bottom: 64px; max-width: 320px;"
    >
      Page not found
    </h1>
    <p
      style="margin-bottom: 48px;"
    >
      Sorry ðŸ˜”, we couldnâ€™t find what you were looking for.
      <br />
      <br />
      <a
        aria-current="page"
        class=""
        href="/"
      >
        Go home
      </a>
      .
    </p>
  </main>
</DocumentFragment>
`;
